terasoluna-gfw-common-libraries/terasoluna-gfw-web-jsp/pom.xml

```diff
@@ -94,13 +94,13 @@
     <!-- == Begin Tomcat == -->
     <!-- Servlet API 3.0 & JSP API 2.2 -->
```

Comment:
- [ ] `Servlet API 4.0 & JSP API 2.3` is correct.

Reply:
fix.
e6026e9aa399fcc7776feaa97897761f5b6a0364